airbyte-integrations/connectors/source-amazon-seller-partner/unit_tests/test_client.py

```diff
@@ -66,21 +66,6 @@ def test_check_connection(mocker):
 
     assert ORDERS_RESPONSE == base_client.check_connection()
 
-
-def test_get_cursor_state():
```

Comment:
why did you get rid of this?

Reply:
I removed this function also not needed anymore